.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az @anirudha
\ No newline at end of file
+*   @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az @anirudha @TackAdam
```

Comment:
CODEOWNERS file showing error 
`Unknown owner on line 2: make sure @TackAdam exists and has write access to the repository`

Reply:
Yes we need to get EE on call to add him and then we should be good

Reply:
Permissions are updated!

Reply:
Thanks @gaiksaya !